### PR TITLE
cleanup(angular): replace wrapAngularDevkitSchematics in library generator

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

We use wrapAngularDevkitSchematics when we create Angular libraries.

## Expected Behavior

Generate the files ourselves

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/angular/src/generators/library/files/base/tsconfig.json__tpl__ [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  "files": [],
+  "include": [],
+  "references": [
+    {
+      "path": "./tsconfig.lib.json"
+    },<% if(unitTesting) { %>
+    {
+      "path": "./tsconfig.spec.json"
+    }<% } %>
```
Comment (leosvelperez) [inline packages/angular/src/generators/library/files/base/tsconfig.json__tpl__:14]: I just realized we shouldn't handle any test-related config & files (except spec files) in the `library` generator. We invoke the `jestProject` generator and that one sets up any file and config needed for a project. It handles this change and others we're doing in this PR.

The same applies to what we already did in the `application` generator. We need to amend that.

File: packages/angular/src/generators/library/files/base/tsconfig.spec.json__tpl__
Comment (leosvelperez) [inline packages/angular/src/generators/library/files/base/tsconfig.spec.json__tpl__:1]: Remove this file

File: packages/angular/src/generators/library/files/base/package.json__tpl__ [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "<%= npmScope %><%= libFileName %>",
```
Comment (leosvelperez) [inline packages/angular/src/generators/library/files/base/package.json__tpl__:2]: `npmScope` might not be defined. It's not a required property of the `nx.json` and we're missing the `/` here.
```suggestion
  "name": "<% if (npmScope) { %><%= npmScope %>/<% } %><%= libFileName %>",
```

File: packages/angular/src/generators/library/lib/create-files.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          project.sourceRoot,
+          `lib/${options.libraryOptions.fileName}.module.ts`
+        )
+      );
+    }
+  }
+
+  if (options.libraryOptions.unitTestRunner === UnitTestRunner.None) {
+    tree.delete(joinPathFragments(project.root, `tsconfig.spec.json`));
+  }
```
Comment (leosvelperez) [inline packages/angular/src/generators/library/lib/create-files.ts:78]: Remove this

File: packages/angular/src/generators/library/files/lib/README.md__tpl__
Comment (leosvelperez) [inline packages/angular/src/generators/library/files/lib/README.md__tpl__:1]: Why are we removing the `README`?

File: packages/angular/src/generators/library/lib/update-project.ts [hunk 4]
```diff
@@ -1,188 +1,13 @@
...
-import { join } from 'path';
 
 export async function updateProject(
   host: Tree,
   options: NormalizedSchema['libraryOptions']
 ) {
-  createFiles(host, options);
   updateProjectTsConfig(host, options);
-  fixProjectWorkspaceConfig(host, options);
   updateNgPackage(host, options);
```
Comment (leosvelperez) [inline packages/angular/src/generators/library/lib/update-project.ts:10]: Do we still need to update `ng-package.json` here? The template should have all that's needed, isn't it?

File: packages/angular/src/generators/library/lib/update-project.ts [hunk 5]
```diff
@@ -1,188 +1,13 @@
...
 import { NormalizedSchema } from './normalized-schema';
 import { updateNgPackage } from './update-ng-package';
-import { join } from 'path';
 
 export async function updateProject(
   host: Tree,
   options: NormalizedSchema['libraryOptions']
 ) {
-  createFiles(host, options);
   updateProjectTsConfig(host, options);
```
Comment (leosvelperez) [inline packages/angular/src/generators/library/lib/update-project.ts:9]: Do we still need to update the tsconfig here? The template should have all that's needed, isn't it?

**Review by leosvelperez — CHANGES_REQUESTED** [review]

File: packages/angular/src/generators/library/files/base/tsconfig.json__tpl__ (hunk 1 above)
Comment (leosvelperez) [reply packages/angular/src/generators/library/files/base/tsconfig.json__tpl__:14]: I created a PR for the application generator to address this https://github.com/nrwl/nx/pull/15731.

File: packages/angular/src/generators/library/files/lib/README.md__tpl__
Comment (Coly010) [reply packages/angular/src/generators/library/files/lib/README.md__tpl__:1]: bad copy i think

Comment (Coly010) [reply packages/angular/src/generators/library/files/lib/README.md__tpl__:1]: actually, it's just been moved

**Review by leosvelperez — APPROVED** [review]

**Pull request merged**